src/TinyCiv.Shared/Events/Client/StartGameClientEvent.cs

```diff
@@ -1,6 +1,8 @@
+using TinyCiv.Shared.Game;
+
 namespace TinyCiv.Shared.Events.Client;
 
 /// <summary>
 /// Use to trigger game start
 /// </summary>
-public record StartGameClientEvent : ClientEvent;
\ No newline at end of file
+public record StartGameClientEvent(MapType MapType = MapType.Test) : ClientEvent;
```

Comment:
@jshimas @Randwarf 

the one who starts the game can choose the map, "MapType.Test" - everything empty as was previously